JULIERAJ/KIDS-FIRST-Front
# Fix/131/sidebar buttons layout dragging

One Line Description
Fix: Resolve sidebar button layout dragging on focus

Test Steps

- Click on sidebar buttons
- Ensure they stay intact and do not drag on focus

Checklist
